ibis/pandas/client.py

```diff
@@ -74,6 +74,8 @@ class PandasTable(ops.DatabaseTable):
 
 class PandasClient(client.Client):
 
+    dialect = None  # defined in ibis.pandas.api
```

Comment:
I define in the API so that we can pickup the implemented operations. There's a better way to do this. I'll do this in a follow up patch.